blueprint: document systemd customization failure

supakeen opened the pull request:
When a service that does not exist on the image being built the build fails. Mention this.

ondrejbudai reviewed (approved):
Better late than never. Love it!

supakeen committed:
blueprint: document systemd customization failure

When a service that does not exist on the image being built the build
fails. Mention this.

ochosi reviewed (approved):
LGTM! Thanks for adding this!
